spring-pulsar-docs/src/main/antora/modules/ROOT/pages/reference/testing-applications.adoc

```diff
@@ -44,3 +44,39 @@ List<Message<String>> messages = PulsarConsumerTestUtil.consumeMessages(consumer
             .until(ConsumedMessagesConditions.atLeastOneMessageMatches("boom"))
             .get();
 ----
+
+== PulsarTestContainerSupport
+
+The `org.springframework.pulsar.test.support.PulsarTestContainerSupport` interface provides a static Pulsar Testcontainer.
+When using Junit Jupiter, the container is automatically started once per test class via `@BeforeAll` annotation.
+
+The following example shows how you can use the container support in a `@SpringBootTest` in conjunction with the previously mentioned `PulsarConsumerTestUtil`.
+
+[source,java,indent=0,subs="verbatim"]
+----
+@SpringBootTest
+class MyApplicationTests implements PulsarTestContainerSupport {
```

Comment:
@jonas-grgt this is cool! I pulled the test util in a demo app and was able to use the container support and consumer test util to do this clean IT

Reply:
Very pleased with this commit! Nice job.